test/integration/Registration.spec.ts

```diff
@@ -0,0 +1,55 @@
+import {Test, TestingModule} from '@nestjs/testing';
+import {AppModule} from '../../src/app.module';
+
+import {
+  CREATE_ADMIN_DTO,
+  CREATE_USER_DTO_TEST_OBJ_WITH_UNMATCHING_PASSWORDS,
+  CREATE_USER_DTO_TEST_OBJ_2
+} from '../constants/users-test.constant';
+
+import {register} from './pages/Registration';
+import {DatabaseService} from '../../src/database/database.service';
+
+describe('Registration', () => {
+  let databaseService: DatabaseService;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+      providers: [DatabaseService]
+    }).compile();
+
+    databaseService = moduleFixture.get<DatabaseService>(DatabaseService);
+  });
+
+  beforeEach(async () => {
+    await page.goto('http:/localhost:8000/signup');
+  });
+
+  it('Registration Success', async () => {
+    const response = await register(page, CREATE_ADMIN_DTO);
+    expect(response).toBe(201);
+  });
+
+  it("Registration failure because password doesn't match", async () => {
+    const response = await register(
+      page,
+      CREATE_USER_DTO_TEST_OBJ_WITH_UNMATCHING_PASSWORDS
+    );
+    expect(response).toBe(400);
```

Comment:
could expect the presence of some page elements here.

Reply:
added expect for page elements